src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -225,15 +225,7 @@ public TimestampInterval processTimeIntervalArgs ( Arguments args )
     }
 
     begin = parseTimeArgument!("-")(begin_arg, reference_time, false);
-
-    if ( isTimeInterval(begin_arg) )
-    {
-        end = parseTimeArgument!("+")(end_arg, time(null), include_end_date);
-    }
-    else
-    {
-        end = parseTimeArgument!("+")(end_arg, begin, include_end_date);
-    }
+    end = parseTimeArgument!("+")(end_arg, begin, include_end_date);
```

Comment:
The commit message "Make end argument to be relative to the begin argument instead of the current timestamp" should describe the rationale for the change.